### PR TITLE
create is not responding

### PR DESCRIPTION
**Pull request by felixevers**
## Description
Fixed bug #13 by outsourcing part_owner function.

## Related Issue
#13 

## Motivation and Context
Fix to get service actions to work.

## How Has This Been Tested?
Yes, with the official [game-server](https://github.com/cryptic-game/server).

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by tanneberger — APPROVED** [review]

File: app/resources/game_content.py [hunk 1]
```diff
@@ -62,3 +62,14 @@ def nmap(data: dict, user: str) -> dict:
             return_data.append(service.public_data())
 
     return {"services": return_data}
+
+
+def part_owner(device: str, user: str) -> dict:
```
Comment (ghost) [inline app/resources/game_content.py]: The return type does not match (it returns True and False, not a dict).

File: app/resources/service.py [hunk 2]
```diff
@@ -108,20 +108,19 @@ def list_services(data: dict, user: str) -> dict:
...
+            services: List[Service] = wrapper.session.query(Service).filter_by(
+                device=data["device_uuid"]).all()
+        elif user == data_return["owner"]:
+            services: List[Service] = wrapper.session.query(Service).filter_by(
+                device=data["device_uuid"]).all()
+        else:
+            return permission_denied
     else:
-        return permission_denied
+        device_does_not_exist
```
Comment (ghost) [inline app/resources/service.py]: A return is missing.

File: app/resources/service.py [hunk 3]
```diff
@@ -163,14 +162,10 @@ def create(data: dict, user: str) -> dict:
...
 @m.user_endpoint(path=["part_owner"])
 def part_owner(data: dict, user: str) -> dict:
-    services: List[Service] = wrapper.session.query(Service).filter_by(device=data["device_uuid"]).all()
-
-    for e in services:
-        if e.part_owner == user and e.running_port is not None and \
-                config["services"][e.name]["allow_remote_access"] is True:
-            return success_scheme
+    if "device_uuid" not in data:
+        invalid_request
```
Comment (ghost) [inline app/resources/service.py]: Here a return is also missing.

**Review by ghost — CHANGES_REQUESTED** [review]
See given code comments.

**Review by tanneberger — DISMISSED** [review]

**Comment by felixevers** [comment]
Fixed in https://github.com/cryptic-game/cryptic-service/commit/4848cc7f0cd40710a97c64489e08cf5663fc7b76

**Review by ghost — APPROVED** [review]

**Review by tanneberger — APPROVED** [review]

**Pull request merged**